code/modules/pda/cart_apps.dm

```diff
@@ -57,75 +61,64 @@
 /datum/data/pda/app/signaller
 	name = "Signaler System"
 	icon = "rss"
-	template = "pda_signaller"
+	template = "pda_signaler"
 	category = "Utilities"
 
 /datum/data/pda/app/signaller/update_ui(mob/user as mob, list/data)
 	if(pda.cartridge && istype(pda.cartridge.radio, /obj/item/integrated_radio/signal))
 		var/obj/item/integrated_radio/signal/R = pda.cartridge.radio
-		data["signal_freq"] = format_frequency(R.frequency)
```

Comment:
If someone enters `1447`, it clamps out at `148.9`, because it divides it inside the TGUI itself. 
![APKMnUGlYw](https://user-images.githubusercontent.com/25063394/94168903-0a4b4800-fe86-11ea-8ced-e2c415ea1760.gif)